kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/ColumnMapping.java

```diff
@@ -307,8 +301,6 @@ private static boolean hasPhysicalName(StructField field) {
     }
 
     private static int getColumnId(StructField field) {
```

Comment:
want to understand why the field id has to be int. We can look at it in the next PRs.